packages/create-sitecore-jss/src/initializers/vue/args.ts

```diff
@@ -0,0 +1,4 @@
+﻿import { BaseArgs } from './../../common/args/base';
+import { ClientAppAnswer } from './../../common/prompts/base';
+
+export interface VueArgs extends BaseArgs, Partial<Omit<ClientAppAnswer, 'fetchWith'>> {}
```

Comment:
I think we want to extend `ClientAppArgs` here instead of `BaseArgs`. Also Vue should support `fetchWith`.